voter-dapp/src/ActiveRequests.js

```diff
@@ -374,6 +431,44 @@ function ActiveRequests({ votingAccount, votingGateway }) {
     }
   };
 
+  const copyStringToClipboard = string => {
```

Comment:
Didn't you say that the clipboard copy didn't work?

Reply:
I got it to work by using the `navigator.writeText` object. Most browsers use the synchronous `document.execCommand` pattern, but Chrome replaced it with the asynchronous `navigator` command after version 66. There's a lengthy discussion [here](https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript) it's pretty fascinating

Reply:
Oh nice!